packages/requisitions/src/SupplierRequisition/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC } from 'react';
+import { useParams } from 'react-router';
+import {
+  TableProvider,
+  createTableStore,
+  useOmSupplyApi,
+  useDocument,
+  useColumns,
+  GenericColumnKey,
+  DataTable,
+  usePagination,
+  useTranslation,
+} from '@openmsupply-client/common';
+
+import { reducer } from './reducer';
+import { getSupplierRequisitionDetailViewApi } from './api';
+import { Toolbar } from './Toolbar';
+import { Footer } from './Footer';
+import { AppBarButtons } from './AppBarButtons';
+import { SidePanel } from './SidePanel';
+
+import { isRequisitionEditable } from '../../utils';
+import { RequisitionLine } from '../../../../mock-server/src/data';
+
+const useDraftSupplierRequisition = () => {
+  const { id } = useParams();
+  const { api } = useOmSupplyApi();
+
+  const { draft, save, dispatch, state } = useDocument(
+    ['requisition', id],
+    reducer,
+    getSupplierRequisitionDetailViewApi(api)
+  );
+
+  const onChangeSortBy = () => {};
+
+  return { draft, save, dispatch, onChangeSortBy, sortBy: state.sortBy };
+};
+
+export const DetailView: FC = () => {
+  const { draft, save, onChangeSortBy, sortBy } = useDraftSupplierRequisition();
+
+  const columns = useColumns<RequisitionLine>(
```

Comment:
Not actually sure if I was meant to be following the ui columns or if they were just for the designs - I did it anyway as they seemed nice - though, I don't claim for them to be 100% complete (so I haven't done translations etc)